### PR TITLE
feat: add ability to delete specific tag

### PR DESCRIPTION
**Pull request by eviishondell**
**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json
Comment (raulkele) [inline package.json:1]: Please don't add or change packages unless they are required for the implementation

File: src/components/Repo/RepoDetails.jsx [hunk 1]
```diff
@@ -242,7 +242,29 @@ function RepoDetails() {
       : `Timestamp N/A`;
     return lastDate;
   };
-
+  const handleDeleteRepo = () => {
+    const confirmed = window.confirm('Are you sure you want to delete this repo?');
```
Comment (raulkele) [inline src/components/Repo/RepoDetails.jsx:246]: We implemend dialogs using MUI's dialog component, not the native window API, check FilterDialog used in the ExplorePage for example.

File: src/components/Repo/RepoDetails.jsx [hunk 2]
```diff
@@ -242,7 +242,29 @@ function RepoDetails() {
...
+    if (confirmed) {
+      const apiUrl = `http://localhost:3000/v2/${name}/manifests/`;
```
Comment (raulkele) [inline src/components/Repo/RepoDetails.jsx:248]: We declare our endpoints inside the api.js file. and send requests using axios not fetch. Check the api.js file and usage of api thought the project for examples.
Also, the backend host is declared in the hosts.js file, don't hardcode it inside code

File: src/components/Repo/RepoDetails.jsx [hunk 3]
```diff
@@ -242,7 +242,29 @@ function RepoDetails() {
...
+      fetch(apiUrl, {
+        method: 'DELETE'
+      })
+        .then((response) => {
+          if (response.status === 202) {
+            // Repo deleted successfully
+            console.log('Repo deleted successfully');
```
Comment (raulkele) [inline src/components/Repo/RepoDetails.jsx:255]: don't leave console logs inside code after finishing implementation

File: src/components/Shared/TagCard.jsx [hunk 4]
```diff
@@ -99,14 +99,53 @@ export default function TagCard(props) {
...
-          Tag
-        </Typography>
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <Typography variant="body1" align="left" className={classes.tagHeading}>
+            Tag
+          </Typography>
+          <IconButton
+            color="primary"
+            style={{ marginLeft: 'auto' }}
+            onClick={() => {
```
Comment (raulkele) [inline src/components/Shared/TagCard.jsx:109]: In general, if your function is longer than 1 line, you want to declare it outside the JSX. Also, same thing abound using the window.confirm api

File: src/components/Shared/TagCard.jsx [hunk 5]
```diff
@@ -99,14 +99,53 @@ export default function TagCard(props) {
...
+              const confirmed = window.confirm('Are you sure you want to perform this action?');
+              if (confirmed) {
+                console.log('Button clicked and confirmed!');
+                console.log(props);
+                console.log('REPONAME: ', repoName);
+                const apiUrl = `http://localhost:3000/v2/${repoName}/manifests/${tag}`;
```
Comment (raulkele) [inline src/components/Shared/TagCard.jsx:115]: api.js for backend requests

File: src/host.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 const hostConfig = {
-  auto: true,
-  default: 'http://localhost:5000'
+  auto: false,
```
Comment (raulkele) [inline src/host.js:2]: don't push this, keep only on local

File: src/setupProxy.js
Comment (raulkele) [inline src/setupProxy.js:1]: I'm not sure what this is but it definitely shouldn't be here

**Review by raulkele — CHANGES_REQUESTED** [review]

**Comment by raulkele** [comment]
Also, you need to squash your commits so you only have 1

File: package.json
Comment (eviishondell) [reply package.json:1]: just reverted it back, didn't realize i changed it sorry!!

**Comment by andaaron** [comment]
Superseded by https://github.com/project-zot/zui/pull/401

**Comment by rchincha** [comment]
@eviishondell Thanks for your PR.

**Comment by eviishondell** [comment]
> @eviishondell Thanks for your PR.

No problem!

**Pull request closed without merging**